tests/testutil/router.go

```diff
@@ -21,10 +21,14 @@ type TestRoute struct {
 
 // Executes "rio route add routename.testing-namespace/{routePath} to {service}"
 // This does not take a domain param, that is setup by default.
+// domain is optional, if empty it will generate a random domain.
 // routePath is optional, if empty it will set only domain.
-func (tr *TestRoute) Add(t *testing.T, routePath string, action string, target TestService) {
+func (tr *TestRoute) Add(t *testing.T, domain string, routePath string, action string, target TestService) {
 	tr.T = t
-	fakeDomain := RandomString(5)
+	fakeDomain := domain
```

Comment:
`fakeDomain` doesn't make sense anymore, just check `domain` and if empty generate into it.